### PR TITLE
fix: Incorrect error response for reset password

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR will fix showing an incorrect error response message when resetting the password.

Fixes https://linear.app/chatwoot/issue/CW-3105/incorrect-error-response-for-reset-password

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

**Loom video**
https://www.loom.com/share/7d32d34bd44f4a678a6f9ef2a64c5122?sid=c6d67478-65d7-441f-bc99-4967a9c76556

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by 01406013368** [comment]
My bonus

**Comment by 01406013368** [comment]
No

File: app/javascript/dashboard/routes/dashboard/settings/profile/ChangePassword.vue [hunk 1]
```diff
@@ -125,10 +125,9 @@ export default {
         });
         this.errorMessage = this.$t('PROFILE_SETTINGS.PASSWORD_UPDATE_SUCCESS');
       } catch (error) {
-        this.errorMessage = this.$t('RESET_PASSWORD.API.ERROR_MESSAGE');
-        if (error?.response?.data?.message) {
-          this.errorMessage = error.response.data.message;
-        }
+        this.errorMessage =
```
Comment (pranavrajs) [inline app/javascript/dashboard/routes/dashboard/settings/profile/ChangePassword.vue:128]: Can't we use `parseAPIErrorResponse` (store/utils/api)?

Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/settings/profile/ChangePassword.vue:129]: Yes, we can use `parseAPIErrorResponse `. Updated the PR https://github.com/chatwoot/chatwoot/pull/9114/commits/1f2ce7c92942f710e469a0130002481a2d075d45

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**